aim/storage/structured/sql_engine/entities.py

```diff
@@ -175,6 +180,78 @@ def remove_tag(self, tag_name: str) -> bool:
         session.flush()
         return tag_removed
 
+    @property
+    def notes(self) -> List[dict]:
```

Comment:
worth adding `notes_obj` method like it is done for tags 🤔

Reply:
maybe... but what would it give us?